javalib/src/main/scala/java/lang/System.scala

```diff
@@ -24,12 +24,75 @@ object System {
   def identityHashCode(x: Object): scala.Int =
     x.cast[Word].hashCode
 
-  def getenv(name: String): String                      = ???
-  def clearProperty(key: String): String                = ???
-  def getProperties(): Properties                       = ???
-  def getProperty(key: String): String                  = ???
-  def getProperty(key: String, default: String): String = ???
-  def setProperty(key: String, value: String): String   = ???
+  private def loadProperties() = {
+    val sysProps = new Properties()
+    sysProps.setProperty("java.version", "1.8")
+    sysProps.setProperty("java.vm.specification.version", "1.8")
+    sysProps.setProperty("java.vm.specification.vendor", "Oracle Corporation")
+    sysProps.setProperty("java.vm.specification.name",
+                         "Java Virtual Machine Specification")
+    sysProps.setProperty("java.vm.name", "Scala Native")
+    sysProps.setProperty("java.specification.version", "1.8")
+    sysProps.setProperty("java.specification.vendor", "Oracle Corporation")
+    sysProps.setProperty("java.specification.name",
+                         "Java Platform API Specification")
+    sysProps.setProperty("line.separator", lineSeparator())
+
+    if (Platform.isWindows) {
+      sysProps.setProperty("file.separator", "\\")
+      sysProps.setProperty("path.separator", ";")
+      val userLang    = fromCString(Platform.windowsGetUserLang())
```

Comment:
Windows doesn't provide locale through any environment variables. User language can be changed in the "Control Panel" and we can fetch it using WinAPI.